README.md

```diff
@@ -53,6 +53,7 @@ NASA | USA | Space | ```PVS``` [NASA Langley Formal Methods Research Program](ht
 [Oracle](https://www.oracle.com/) | Redwood Shores, CA, USA | Enterprise software, Cloud computing, Computer hardware | `ACL2` ([Proving Theorems about Java and the JVM with ACL2](https://www.cs.utexas.edu/users/moore/publications/marktoberdorf-02/index.html))
 [Particular Software](https://particular.net/) | | | ```TLA+``` [TLA+ Specifications for NServiceBus](https://github.com/tmasternak/NServiceBus.ModelChecking)
 PingCAP | | | ```TLA+``` [TLA+ in TiDB](https://github.com/pingcap/tla-plus)
+[Prover Technology](https://www.prover.com/) | Europe | Railway | `Model checking`
```

Comment:
Are there any public resources that prove using a model checking?
Papers, slides etc.

Reply:
The blog has many articles about this (e.g. https://www.prover.com/verification-validation/psl-best-in-class-formal-verification-performance/ and https://www.prover.com/modeling/using-hll-to-solve-sudoku-puzzles/), but I can see if I can dig up some academical papers.